README.md

```diff
@@ -848,70 +730,71 @@ A `Receipt` is an attestation of the [Result] and requested [Effect]s by a [Task
 
 Receipts MUST use the same version as the invocation that they contain.
 
-## 8.1 Schema
+## 8.1 Receipt
 
-```ipldsch
-type Receipt struct {
-  ran     &Invocation
-
-  # output of the invocation
-  out     Result
+```
+type Outcome struct {
+  ran     &Invocation # Invocation this is a receipt for
 
-  # Effects to be performed
-  fx      Effects
+  out     Result # Output of the invocation
+  fx      Effects # Effects to be enqueued
 
-  # All the other metadata
-  meta    {String: any}
+  meta    {String : Any} # All the other metadata
 
-  # Principal that issued this receipt.
-  # If omitted issuer is inferred from 
-  # the invocation task audience.
+  # Principal that issued this receipt. If omitted issuer is
+  # inferred from the invocation task audience.
   iss     optional Principal
 
-  # When issuer is different from executor
-  # this MUST hold a UCAN delegation chain 
-  # from executor to the issuer.
-  # This field SHOULD be omitted executor is an issuer.
+  # When issuer is different from executor this MUST hold a UCAN
+  # delegation chain from executor to the issuer. This should be 
+  # omitted when the executor is the issuer.
   prf     [&UCAN]
-
-  # Signature from the `iss`.
-  s       Varsig
+  
+  sig     Varsig
 }
 ```
 
-## 8.2 Fields
-
-### 8.2.1 Ran Invocation
+### 8.1.1 Ran Invocation
 
 The `ran` field MUST include a link to the [Invocation] that the Receipt is for.
 
-### 8.2.2 Output
+### 8.1.2 Output
 
 The `out` field MUST contain the value output of the invocation in [Result] format.
 
-### 8.2.3 Effect
+### 8.1.3 Effect
 
 The OPTIONAL `fx` field, if present MUST be set to the caused [Effect]. The [Executor] SHOULD invoke contained [Task] to progress a workflow execution.
 
 If `fx` does not contain OPTIONAL `join` field, it denotes completion of the current execution thread.
 
-### 8.2.4 Metadata Fields
+### 8.1.4 Metadata Fields
 
 The OPTIONAL metadata field MAY be omitted or used to contain additional data about the receipt. This field MAY be used for tags, commentary, trace information, and so on.
 
-### 8.2.5 Receipt Issuer
+### 8.1.5 Receipt Issuer
 
 The OPTIONAL `iss` field, if present MUST contain the DID of the [Executor] delegate that signed it. If field is present, delegation from [Executor] MUST be included in the `prf` field.
 
 If `iss` field is omitted, Receipt MUST be signed by the [Executor].
 
-### 8.2.6 Proofs
+### 8.1.6 Proofs
```

Comment:
Hasn't Executor been already authorized to perform excution of an Invocation via the Invocation's `auth` field?

Reply:
Ah, I see, Receipt Proofs are for Executor's delegate to proof it's been authorized by Executor